iron-ajax.html

```diff
@@ -302,7 +305,8 @@
       /**
        * By default, these events do not bubble largely because the `error` event has special
        * meaning in the window object. Setting this attribute will cause iron-ajax's request,
-       * response, and error events to bubble to the window object.
+       * response, and error events to bubble to the window object. It will also cause
+       * iron-ajax to emit duplicate events prefixed with "iron-ajax-".
```

Comment:
Please reword to take into account that firing will happen regardless of value :beers: